docs/scripts_md/get_dicom_files.md

```diff
@@ -22,16 +22,20 @@ Available options are:
            the scan type name is not used to determine which DICOM files to extract. This option
            must be used if no patient name patterns were specified via `-n` (see above).
 
-\-d       : extract the files in directory `<dir_argument>/get_dicom_files.pl.<UNIX_process_number>`
-           For example with `-d /data/tmp`, the DICOM files will be extracted in 
-           `/data/tmp/get_dicom_files.pl.67888` (assuming 67888 is the process number). 
-           By default, dir\_argument is set to the value of the environment variable `TMPDIR`. Note 
-           that the directory argument has to exist and be writeable, otherwise the script will issue
-           an error and abort.
+\-d       : extract the files in directory `<dir_argument>/get_dicom_files.pl.<random_string>`
+           For example with `-d /data/tmp`, the DICOM files might be extracted in 
+           `/data/tmp/get_dicom_files.pl.n1d4`. By default, dir\_argument is set to the value of
+           the environment variable `TMPDIR`. Since the UNIX program `tar` has known limitations 
+           with NFS file systems (incorrect reports of files that changed while they are archived), the
+           argument to `-d` should not be a directory that resides on an NFS mounted file system.
+           Failure to do so might result in `get_dicom_files.pl` failing.
 
 \-o       : basename of the final `tar.gz` file to produce, in the current directory (defaults to 
            `dicoms.tar.gz`).
 
+\-p       : when storing a DICOM file in the archive, use the PSCID of the candidate as one of the 
```

Comment:
Random question, wouldn't be easier to include both PSCID and DCCID by default? Or do you think that usually users would only want one or the other? I don't have a strong opinion on this. Was just wondering

Reply:
I now provide more than one way of naming the subdirectory (see option `-id`).